docs/content/configuration/creating_volumestorageclass.md

```diff
@@ -56,6 +56,7 @@ Use the `SpaceEfficiency` parameters for each storage system, as defined in the
       parameters:
         pool: demo-pool
         io_group: demo-iogrp             # Optional.
+        volume_group: demo-volumegroup             # Optional.
```

Comment:
```suggestion
        volume_group: demo-volumegroup  # Optional.
```
(should be the same as in [here](https://github.com/IBM/ibm-block-csi-driver/blob/0de102723dd808cf55e95ce92035cc6b89e9f5d1/deploy/kubernetes/examples/demo-storageclass.yaml#L9))

Reply:
updated spacing. it just got copied wrong